seeds/06_migrate_from_source.js

```diff
@@ -0,0 +1,57 @@
+require('dotenv').config()
```

Comment:
Is the idea of this script that you're cloning agencies and users from CARES/CRF reporting?

I assume there will be some overlap between agencies involved in CARES and in ARPA, but I wonder if we should run that by the states before cloning everything wholesale.